lib/app_profiler/profile/stackprof.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module AppProfiler
+  class StackprofProfile < AbstractProfile
+    FILE_EXTENSION = ".stackprof.json"
```

Comment:
How would this handle legacy profile names without this extension?

Reply:
Right. Sounds good, thanks! I'm going to focus on reviewing this one first. Perhaps we can punt this to the other PR if it only needed for that?